src/main/resources/assets/api-docs.yaml

```diff
@@ -1550,6 +1550,9 @@ components:
         match:
           type: boolean
           description: True if the entities match, false otherwise.
+        abstain:
+          type: boolean
+          description: Nullable boolean will return true if the decision is abstain
```

Comment:
Interesting that we need this in two places, we should probably combine. That can be a separate PR though.